src/java/org/apache/cassandra/cql3/statements/ModificationStatement.java

```diff
@@ -67,7 +67,7 @@
 /*
  * Abstract parent class of individual modifications, i.e. INSERT, UPDATE and DELETE.
  */
-public abstract class ModificationStatement implements CQLStatement
+public abstract class ModificationStatement implements CQLStatement, KeyspaceStatement
```

Comment:
but `KeyspaceStatement` extends `CQLStatement`

perhaps we don't need `KeyspaceStatemetn` to extend it?